src/main/java/io/nats/client/PushSubscribeOptions.java

```diff
@@ -35,6 +35,10 @@ public String getDeliverSubject() {
         return consumerConfig.getDeliverSubject();
     }
 
+    public static PushSubscribeOptions source(String stream) {
+        return new Builder().stream(stream).build();
```

Comment:
I landed on using source in this shortcut builder. open for comments.

Reply:
I'd suggest coordinating with @wallyqs on naming.  Source works for me.

Reply:
I think the go client we have `nats.BindStream()` to set the stream name for mirrors, would this be similar?

```go
js.SubscribeSync("origin", nats.BindStream("m1"))
```

Reply:
changing to `bind` instead of `source`